src/components/small-item-table/index.js

```diff
@@ -338,7 +339,7 @@ function SmallItemTable(props) {
                     formattedItem.barterPrice = getCheapestBarter(itemData, formattedItem.barters);
 
                     if (!itemData.avg24hPrice || formattedItem.barterPrice.price < itemData.avg24hPrice) {
-                        formattedItem.pricePerSlot = Math.floor(formattedItem.barterPrice.price / itemData.itemProperties.grid?.totalSize);
+                        formattedItem.pricePerSlot = Math.floor(formattedItem.barterPrice.price / (itemData.itemProperties.grid?.totalSize - itemData.slots));
```

Comment:
Don't we need a divide by zero protection here too?

Reply:
You're right, thanks for pointing this out. I've made the change in my latest commit.